md_api/src/main/java/com/wipro/ats/bdre/md/api/HaltJob.java

```diff
@@ -81,8 +88,7 @@ public HaltJobInfo execute(String[] params) {
             //The HaltJob completes even if sending message fails
             try {
                 BasicConfigurator.configure();
-                StatusNotification statusNotification = new StatusNotification(haltMessage, MDConfig.getProperty("status-notification.halt-queue"));
-                LOGGER.info(statusNotification.toString());
+                new StatusNotification(haltMessage, MDConfig.getProperty("status-notification.halt-queue"));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either remove this useless object instantiation of class "StatusNotification" or use it [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1848)